metrics/storage/fio_job.sh

```diff
@@ -206,6 +215,9 @@ function main()
 		die "Unknown runtime: $RUNTIME"
 	fi
 
+	# Drop the host page cache
+	sudo echo 3>/proc/sys/vm/drop_caches
```

Comment:
Metrics jenkins is failing on:
```
storage/fio_job.sh: line 219: /proc/sys/vm/drop_caches: Permission denied
```

You may need to do something like:
```
sudo bash -c "echo 0 > ${KSM_ENABLE_FILE}"
```
taken from https://github.com/clearcontainers/tests/blob/master/.ci/run_metrics_ci.sh#L63